Assets/Materials/LoadingLogo/LoadingLogoCircle.mat

```diff
@@ -0,0 +1,43 @@
+%YAML 1.1
```

Comment:
![grafik](https://user-images.githubusercontent.com/11379989/204027372-50a9bab1-fa5b-415d-b908-5b55d9627f22.png)
![grafik](https://user-images.githubusercontent.com/11379989/204027515-13e1a863-6327-47b4-a4e1-7d93ed857728.png)
The Colors are a bit dull